### DISCUSSION TITLE
formula input error

### DISCUSSION THREAD

**Discussion by ARC-MX** [James-Yu/LaTeX-Workshop#3299] [Q&A]
~bug

You can compile a TeX document manually?: Yes

## Describe the bug [Required]
the input formula is :
![image](https://user-images.githubusercontent.com/34829404/169682440-19aeb6ab-40f8-461a-b249-602060a53e81.png)

but the "-" become "*" like this:
![image](https://user-images.githubusercontent.com/34829404/169682481-e267819e-a723-4fb8-b7f8-82358c19e6f0.png)
 I don't know what happend 

OS: Ubuntu 20.04
vscode : 1.66.2
LaTeX Workshop v8.26.0
TeX 3.141592653 (TeX Live 2022)
kpathsea version 6.3.4
Copyright 2022 D.E. Knuth.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the TeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the TeX source.
Primary author of TeX: D.E. Knuth.

**Comment by James-Yu** [comment]
Not related to this extension.
